### PR TITLE
Update shared runtime scenario to use GitHub page hosting

### PR DESCRIPTION
**Pull request by davidchesnut**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no                                |
| New feature?    | no                                |
| New sample?     | no                                |
| Related issues? | none |

## What's in this Pull Request?

Update the excel-shared-runtime-scenario sample to use GitHub page hosting. This removes all package dependencies, and also fixes all security alerts that were in those packages. Should make the sample much easier to maintain going forward.

### CODE REVIEW

File: Samples/excel-shared-runtime-scenario/README.md
Comment (AlexJerabek) [inline Samples/excel-shared-runtime-scenario/README.md:37]: I believe this is the preferred branding.
```suggestion
- Excel on the web
```

File: Samples/excel-shared-runtime-scenario/manifest.xml
Comment (AlexJerabek) [inline Samples/excel-shared-runtime-scenario/manifest.xml:17]: Should these be pointing to officedev instead of davidchesnut?

File: Samples/excel-shared-runtime-scenario/README.md
Comment (AlexJerabek) [inline Samples/excel-shared-runtime-scenario/README.md:66]: Same branding comment.
```suggestion
You can run this sample in Excel on the web. The add-in web files are served from this repo on GitHub.
```

File: Samples/excel-shared-runtime-scenario/README.md
Comment (AlexJerabek) [inline Samples/excel-shared-runtime-scenario/README.md:152]: Suggestion for brevity.
```suggestion
2. Use a tool such as openssl to generate a self-signed certificate for the web server. Move the cert.pem and key.pem files to the webworker-customfunction folder for this sample.
```

File: Samples/excel-shared-runtime-scenario/README.md
Comment (AlexJerabek) [inline Samples/excel-shared-runtime-scenario/README.md:70]: Nit about file types.
```suggestion
1. Choose **Excel**, and then open a new workbook.
```

File: Samples/excel-shared-runtime-scenario/README.md
Comment (AlexJerabek) [inline Samples/excel-shared-runtime-scenario/README.md:74]: There technically two manifest files here: the standard and one for localhost. Is that intentional?

**Review by AlexJerabek — APPROVED** [review]
Approving after experiencing the working sample on Excel on the web.

File: Samples/excel-shared-runtime-scenario/manifest.xml
Comment (davidchesnut) [reply Samples/excel-shared-runtime-scenario/manifest.xml:17]: Yes, I have to update this after the merge.

File: Samples/excel-shared-runtime-scenario/README.md
Comment (davidchesnut) [reply Samples/excel-shared-runtime-scenario/README.md:74]: Yes, later in the readme we explain how to set up your own localhost if you prefer.

**Pull request merged**
